D1673/P1673.md

```diff
@@ -713,6 +713,8 @@ toc: true
 
     * fix unary plus and well formed
 
+    * remove _`proxy-reference`_ , _`scaled_scalar`_ and _`conjugated_scalar`_
+
```

Comment:
I can't add a suggestion too far away from this, but should we also consider renaming `conjugate_accessor` to `conjugated_accessor`, as it better matches the function name `conjugated`?